shared/cmd/config.go

```diff
@@ -20,67 +20,62 @@ var sharedConfig *Flags
 func Get() *Flags {
 	if sharedConfig == nil {
 		return &Flags{
-			MaxRPCPageSize: params.BeaconConfig().DefaultPageSize,
+			MaxRPCPageSize:     params.BeaconConfig().DefaultPageSize,
+			CustomGenesisDelay: params.BeaconConfig().GenesisDelay,
 		}
 	}
 	return sharedConfig
 }
 
-// Init sets the global config equal to the config that is passed in.
-func Init(c *Flags) {
+// OverrideConfig sets the global config equal to the config that is passed in.
+func OverrideConfig(c *Flags) {
 	sharedConfig = c
 }
 
 // InitWithReset sets the global config and returns function that is used to reset configuration.
 func InitWithReset(c *Flags) func() {
 	resetFunc := func() {
-		Init(&Flags{})
+		OverrideConfig(&Flags{})
 	}
-	Init(c)
+	OverrideConfig(c)
 	return resetFunc
 }
 
 // ConfigureBeaconChain sets the global config based
 // on what flags are enabled for the beacon-chain client.
 func ConfigureBeaconChain(ctx *cli.Context) {
 	cfg := newConfig(ctx)
-	delay := params.BeaconConfig().GenesisDelay
```

Comment:
please revert all the genesis delay changes, this breaks how we run the node. The default genesis delay is now zero...

Reply:
ok so it seems like it is set here:
https://github.com/prysmaticlabs/prysm/blob/0178c2b2922a323b59b94c9afa3ba1760fd0842d/shared/cmd/config.go#L24 , although this seems a bit brittle, if `sharedConfig` is already initialized we would ignore it. Can you double check it doesn't break genesis for our testnets ?

Reply:
Please check newConfig(), the defaults are now set through there. I can confirm with a test.

Reply:
Added a test to confirm the default value @nisdas please check

Reply:
I also synced a node to head with these changes :+1: